controls/radrichtextbox/document-elements/table/formatting-tables.md

```diff
@@ -0,0 +1,148 @@
+
+                if (row == 0)
+                {
+                    currnetCell.Borders = new TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder);
+                }
+                else
+                {
+                    currnetCell.Borders = new TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder1);
+                }
+
+                if (col % 2 == 0)
+                {
+                    currnetCell.Background = Color.FromArgb(117, 220, 220, 220);
+                }
+
+            }
+        }
+
+        SetTableData(table);
+
+        var editor = new RadDocumentEditor(this.radRichTextBox.Document);
+        editor.InsertTable(table);
+    }
+
+{{endregion}}
+
+#### __[VB.NET] Generate a table and set its borders and background__
+
+{{region radrichtextbox-document-elements-formatting-tablesVB_0}}   
+
+	Public Sub SetBorders()
+		Dim table = New Table(10, 5)
+
+		Dim blackBorder = New Telerik.Windows.Documents.Model.Border(2, BorderStyle.Single, Colors.Green)
+		Dim grayBorder = New Telerik.Windows.Documents.Model.Border(3, BorderStyle.Single, Colors.Gray)
+		Dim grayBorder1 = New Telerik.Windows.Documents.Model.Border(1, BorderStyle.Single, Colors.Gray)
+
+		Dim rows = table.Rows.ToArray()
+
+		For row As Integer = 0 To table.Rows.Count - 1
+			Dim currentRow = rows(row)
+			currentRow.Height = 50
+			Dim cells = currentRow.Cells.ToArray()
+
+			For col As Integer = 0 To currentRow.Cells.Count - 1
+				Dim currnetCell = cells(col)
+				currnetCell.TextAlignment = Telerik.Windows.Documents.Layout.RadTextAlignment.Center
+				currnetCell.VerticalAlignment = Telerik.Windows.Documents.Layout.RadVerticalAlignment.Center
+
+				If row = 0 Then
+					currnetCell.Borders = New TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder)
+				Else
+					currnetCell.Borders = New TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder1)
+				End If
+
+				If col Mod 2 = 0 Then
+					currnetCell.Background = Color.FromArgb(117, 220, 220, 220)
+				End If
+
+			Next col
+		Next row
+
+		SetTableData(table)
+
+		Dim editor = New RadDocumentEditor(Me.radRichTextBox.Document)
+		editor.InsertTable(table)
+	End Sub
+
+
+{{endregion}}
+
+>caption Figure 1: Merge cells in code
+
+![formatting-tables001](images/formatting-tables001.png)
+
+
+## Using the StyleName Property
+
+You can use the __StyleName__ property to set one of the predefined table styles.
+
+#### __[C#] Set the StyleName__
+
+{{region radrichtextbox-document-elements-formatting-tables_1}}   
+
+    var table = new Table(10, 5);
+    table.StyleName = RadDocumentDefaultStyles.TableGridLightStyleName;
+
+{{endregion}}
+
+#### __[VB.NET] Set the StyleName__
+
+{{region radrichtextbox-document-elements-formatting-tablesVB_1}}   
+    
+    Dim table = New Table(10, 5)
+    table.StyleName = RadDocumentDefaultStyles.TableGridLightStyleName
+
+{{endregion}}
+
+>caption Figure 2: Use predefined styles
```

Comment:
Can we use another style here? I am not sure the example will be clear enough as the styling applied by the TableGrid Light style is not so visible